package.json

```diff
@@ -24,7 +24,7 @@
 		"clean-extensions": "rm -rf _inc/blocks/ ",
 		"install-if-deps-outdated": "yarn check 2> /dev/null || yarn install --check-files --production=false",
 		"distclean": "rm -rf node_modules && yarn clean",
-		"build": "yarn install-if-deps-outdated && yarn clean && yarn build-client && yarn build-php && yarn build-extensions",
+		"build": "yarn install-if-deps-outdated && yarn clean && yarn build-php && yarn build-client && yarn build-extensions",
```

Comment:
quick note to say the order needed to change so that the php packages are in place before we build the client, which now depends on assets in those packages.